src/frameworks/drupal7/simplesaml.md

```diff
@@ -0,0 +1,115 @@
+# SimpleSAML
+
+SimpleSAMLphp is a library for authenticating a PHP-based application against a SAML server, such as Shibboleth.  Although Drupal has modules available to authenticate using SimpleSAML some additional setup is required.
+
+The following setup assumes you're using the `drupal` build flavor and building your site with Drush Make.  If not, you may need to adjust some paths in the configuration but the basics are the same.
+
+## Download the library
+
+First, download the 3rd party [SimpleSAMLphp library](https://simplesamlphp.org/download).  When you unpack the `tar.gz` file it will contain a directory named `simplesamplephp-???`, where the `???` is the version number of the library.  Place that directory at the root of your application, as a sibling of your `.platform.app.yaml` file, named `simplesamplephp`.  (The directory name doesn't really matter but removing the version number means that it won't change in future updates.) 
+
+The `drupal` build flavor will move that directory to the `public/sites/default/` directory during build.  The rest of the configuration is based on that behavior.
+
+## Expose the SimpleSAML endpoint
+
+The SimpleSAML library's `www` directory needs to be publicly accessible.  That can be done by mapping it directly to a path in the Application configuration.  Add the following block to the `web.locations` section of `.platform.app.yaml`:
+
+```yaml
+ web:
+    locations:
+        '/simplesaml':
+            root: 'public/sites/default/simplesamlphp/www'
+            allow: true
+            scripts: true
+            index:
+                - index.php
+```
+
+That will map all requests to `example.com/simplesaml/` to the `simplesamlphp/www` directory, allowing static files there to be served, PHP scripts to execute, and defaulting to index.php.
+
+## Install the `simpleSAMLphp Authentication` module
+
+You will need to install the [simpleSAMLphp Authentication](https://www.drupal.org/project/simplesamlphp_auth) module.  If using Drush Make then the easiest way to do so is simply to add the following line to your `project.make` file:
+
+```ini
+projects[simplesamlphp_auth][version] = 2.0-alpha2
+```
+
+(Adjust the version to whatever is current.)
+
+Much of the module configuration will depend on your Identity Provider (IdP).  However, the module also need to know the location of your `simplesamlphp_auth` module.  The easiest way to set it is to include the following at the end of your `settings.platformsh.php` file:
+
+```php
+// Set the path for the SimpleSAMLphp library dynamically.
+if (isset($_ENV['PLATFORM_APP_DIR'])) {
+  $conf['simplesamlphp_auth_installdir'] = dirname(realpath(__FILE__)) . '/simplesamlphp';
```

Comment:
Hm, a valid point.  This line probably is safe to run even on local, precisely because it's relative.  I'll just remove the conditional entirely.  Good catch!